db/migrate/20231006064505_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :UserId
+      t.references :post, null: false, foreign_key: true
+      t.string :PostId
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] For the comments table as well I'd suggest adding the `text` attribute. 
- [ ] Please note that whenever you use  `t.references :column_name` rails automatically generates an attribute with a `column_id` so there's no need for specifying it explicitly as such I'd suggest removing the code at lines `5` and `7` as they're redundant.